index.html

```diff
@@ -29,6 +29,7 @@
     />
```

Comment:
- [ ] Whilst your branch has no conflicts, your Linters are not running, kindly fix this issue.

![image](https://user-images.githubusercontent.com/81932478/169662439-76d5a596-7e81-49c3-af34-85be5ca4c84c.png)
#### *The image above shows not linters